Makefile

```diff
@@ -35,7 +35,7 @@ clean:
 $(RUNTIME_TOOLS_LINK):
 	ln -sf $(CURDIR) $(RUNTIME_TOOLS_LINK)
 
-.PHONY: test .gofmt .govet .golint
```

Comment:
Well, I think this is not a bug or wrong content. 
Putting them into .PHONY is a way to defend in case of there is `.gofmt`, `.govet` and so son which will break make process.
As there is not any one in project directory, removing them or not is got a big deal.
But noting `fix` in comment is not suitable, `cleanup` will be better.